pkgs/tools/networking/dae/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, llvmPackages
+, fetchFromGitHub
+, buildGoModule
+}:
+buildGoModule rec {
+  pname = "dae";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "daeuniverse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-EoStRmyYOtvG5ejtvHWNe9IIeE77hqp1OXBhRdxCYHs=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-vxHufE3538l6zIcozFcrNhl+2sG1PtzkVxC0NyL3WMU=";
+
+  proxyVendor = true;
+
+  GOFLAGS = "-buildvcs=false";
```

Comment:
`GOFLAGS` is shadowed by https://github.com/NixOS/nixpkgs/blob/a5aeb5016dec640058180411033f3c6409153646/pkgs/build-support/go/module.nix#L176
So it wasn't needed in the first place?

Reply:
> So it wasn't needed in the first place?

Seems yes.